src/components/keyboardAvoidingView.rei

```diff
@@ -0,0 +1,59 @@
+let make:
+  (
+    ~accessibilityLabel: string=?,
+    ~accessible: bool=?,
+    ~hitSlop: Types.insets=?,
+    ~onAccessibilityTap: unit => unit=?,
+    ~onLayout: RNEvent.NativeLayoutEvent.t => unit=?,
+    ~onMagicTap: unit => unit=?,
+    ~responderHandlers: Types.touchResponderHandlers=?,
+    ~pointerEvents: [ | `auto | `boxNone | `boxOnly | `none]=?,
+    ~removeClippedSubviews: bool=?,
+    ~style: Style.t=?,
+    ~testID: string=?,
+    ~accessibilityComponentType: [
+                                   | `button
+                                   | `none
+                                   | `radiobutton_checked
+                                   | `radiobutton_unchecked
+                                 ]
+                                   =?,
+    ~accessibilityLiveRegion: [ | `assertive | `none | `polite]=?,
+    ~collapsable: bool=?,
+    ~importantForAccessibility: [ | `auto | `no | `noHideDescendants | `yes]=?,
+    ~needsOffscreenAlphaCompositing: bool=?,
+    ~renderToHardwareTextureAndroid: bool=?,
+    ~accessibilityTraits: list(
+                            [
+                              | `adjustable
+                              | `allowsDirectInteraction
+                              | `button
+                              | `disabled
+                              | `frequentUpdates
+                              | `header
+                              | `image
+                              | `key
+                              | `link
+                              | `none
+                              | `pageTurn
+                              | `plays
+                              | `search
+                              | `selected
+                              | `startsMedia
+                              | `summary
+                              | `text
+                            ],
+                          )
+                            =?,
+    ~accessibilityViewIsModal: bool=?,
+    ~shouldRasterizeIOS: bool=?,
+    ~keyboardVerticalOffset: float=?,
+    ~behavior: [ | `height | `position | `padding]=?,
```

Comment:
Could we use the actual type defined in the module here instead of having to keep the two in sync.

Reply:
I think I need an explanation because it's the interface file, and the `behavior` type is not defined here 🤔

Reply:
Everything created in the re file should be available in the rei file as well. But I don't think it's a huge issue. This can always be generated so it will probably never be manually kept in sync anyways.